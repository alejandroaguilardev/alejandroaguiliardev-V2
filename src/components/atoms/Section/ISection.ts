import { ReactNode } from "react";

export interface ISection {
	children: ReactNode;
}
