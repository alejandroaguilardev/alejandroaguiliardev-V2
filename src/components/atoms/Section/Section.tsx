import { Box } from "@mui/material"
import { ISection } from "./ISection"

export const Section = ({ children }: ISection) => {
    return (
        <Box sx={{ my: 20 }}>
            {children}
        </Box>
    )
}
