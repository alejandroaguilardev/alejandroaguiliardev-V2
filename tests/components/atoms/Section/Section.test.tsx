import { render, screen } from '@testing-library/react';
import { Section } from '../../../../src/components/atoms/Section/Section';

describe("Section", ()=> {
    test("toMacthSnapshot", ()=>{ 
        const {container} = render(<Section>
            <h1>Section</h1>
        </Section>)

        expect(container).toMatchSnapshot();
        expect(screen.getByText("Section")).toBeTruthy();
    })
})