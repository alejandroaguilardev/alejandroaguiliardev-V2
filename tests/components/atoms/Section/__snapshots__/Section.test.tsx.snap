// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section toMacthSnapshot 1`] = `
<div>
  <div
    class="MuiBox-root css-oomdnv"
  >
    <h1>
      Section
    </h1>
  </div>
</div>
`;
